react/react_events/README.md

```diff
@@ -10,27 +10,27 @@
 
 ## Objectives
 
-- Understand what `state` does in a React application
-- Instantiate `state` in a constructor
+- Understand what state does in a React application
+- Instantiate `state` in a component's constructor
 - Call `setState` to update state
-- Refer to state using the `{}` JSX syntax
+- Refer to and utilize state in a component's render function
 
-# 1.  Introduction
-
-In the previous lesson, we saw how to create a static web page using React components.  In this lesson, we will start to review the strategy that React uses to handle user input, using the `state` property.
+# Introduction
 
+In the previous lesson, we saw how to create a static web page using React components and props. In this lesson, we will start to review the strategy that React uses to handle changing data, user input, and visual updates. For nearly anything that requires information to update on the page, we use the `state` property.
```

Comment:
I want to get away from "user actions" as a primary descriptor because state doesn't exclusively update in response to user action. That said, editing this for clarity.